README.md

```diff
@@ -4,11 +4,11 @@ A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
 deploying and managing [Tmate self-hosted server](https://tmate.io/). Tmate is an
 open source terminal multiplexer, providing instant terminal sharing capabilities.
 
-Tmate is a terminal multiplexer that allows remote terminal sharing. It enables users to share
-their terminal session with other users over the internet, allowing them to collaborate, provide
-technical support, or demonstrate commands and procedures in real-time.
+Tmate enables users to share their terminal session with other users over the internet, allowing
```

Comment:
From [previous PR](https://github.com/canonical/tmate-ssh-server-operator/pull/2)
[@jdkandersson] Consider whether the repetitiveness can be reduced here.